Scripts/Spice.py

```diff
@@ -17,14 +19,185 @@ def compute_view_frustrum(instrument):
 
 def spacecraft_position_and_orientation(spacecraft, point):
     """
-    stub
+    Look under SPK Kernel for now
     :param spacecraft: Data structure that models the spacecraft.
     :param point: point in time.
     :return:
     """
     pass
 
-
 """
 Create Spice Wrappers Here:
-"""
\ No newline at end of file
+"""
+
+###########
+# Kernels #
+###########
+
+class Kernel(enum.Enum):
```

Comment:
Let's name this something more explicit like `KernelType` so that it is more clear that this is an enum rather than an object corresponding to a single Kernel.
```suggestion
class KernelType(enum.Enum):
```